hear_streaming:
  ros__parameters:
    STT_SERVER_IP: '127.0.0.1:50051'
    STT_ACTION_SERVER_NAME: 'stt_streaming'
    AUDIO_TOPIC: '/rawAudioChunk'
    DEFAULT_HOTWORDS: REPLACE
